src/AppBundle/Controller/AdminController.php

```diff
@@ -77,6 +79,24 @@ public function ordersAction(Request $request)
         );
     }
 
+    /**
+     * @Route("/admin/orders/{id}.pdf", name="admin_order_invoice", requirements={"id" = "\d+"})
```

Comment:
I think this feature should be accessible to everyone (restaurants, customers too in `/profile`).
Lots of customers will need the pdf for their works invoiving.

Reply:
Yep, this is in my TODOLIST